integrations/dd-sdk-android-glide/src/main/kotlin/com/datadog/android/glide/DatadogGlideModule.kt

```diff
@@ -49,6 +50,10 @@ constructor(
     private val sampleRate: Float = DEFAULT_SAMPLE_RATE
 ) : AppGlideModule() {
 
+    private val firstPartyHostsWithHeaderType = firstPartyHosts.associateWith {
```

Comment:
the only problem I see with this is that some users will not know what to use for their tracing header types values so we might want to provide a Public Constant: `DEFAULT_TYPES` here ?

Reply:
why then we need to provide association if `setOf(TracingHeaderType.DATADOG, TracingHeaderType.TRACECONTEXT)` is what already set in the SDK by default? Shouldn't we have a public API for both `firstPartyHosts` and `firstPartyHostsWithHeaderType` as we have now?

Reply:
we could but the way I see it `firstPartyHosts` is mandatory (not optional) so it will have to be in the `Builder` constructor. If we want to introduce both APIs and if a user wants to provide custom headerTypes for each host it will look like this:
```
val builder = Builder(tracedHosts).setFirstPartyHostsWithHeaderTypes(firstPartyHostsWithHeaderTypes)
```
I do not like that.

Reply:
otherwise we could have:
```
val builder = Builder.()setFirstPartyHostsWithHeaderTypes(firstPartyHostsWithHeaderTypes)
val builder = Builder().setFirstPartyHosts()
```
But this will mean that some users will forget to set this up in the builder and will create support issues.